local-cli/library/library.js

```diff
@@ -71,7 +57,15 @@ function _library(argv, config, resolve, reject) {
   console.log('Next Steps:');
   console.log('   Link your library in Xcode:');
   console.log('   https://facebook.github.io/react-native/docs/linking-libraries-ios.html#content\n');
-  resolve();
 }
 
-module.exports = library;
+module.exports = {
+  name: 'new-library',
+  func: library,
+  description: 'generates a native library bridge',
+  options: [{
+    command: '--name <string>',
+    description: 'name of the library to generate',
+    default: null,
```

Comment:
This is actually a `commander` bug, `name` is defined on its prototype, so if we don't provide a default value, it's a function. Should be reported or the name itself should be changed to something else.